File: Daily Test/Java/Seven Segment Display To Binary 10.9.2025.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        String[] d = {
            "111110", "011000", "1101101", "1111001", "0110011",
            "1011011", "1011111", "1110000", "1111111", "1111011"
        };
        Scanner sc = new Scanner(System.in);
        String N = sc.nextLine().trim();
        StringBuilder res = new StringBuilder();
        for (char c : N.toCharArray())
            res.append(d[c-'0']);
        int num = Integer.parseInt(res.toString(), 2);
        System.out.println(num);
    }
}

File: Daily Test/C++/Seven Segment Display To Binary 10.9.2025.cpp
#include <iostream>
#include <string>
using namespace std;

string d[] = {
    "111110", "011000", "1101101", "1111001", "0110011",
    "1011011", "1011111", "1110000", "1111111", "1111011"
};

int main() {
    string N, res = "";
    cin >> N;
    for (char c : N)
        res += d[c-'0'];
    int num = 0;
    for (char c : res)
        num = num * 2 + (c - '0');
    cout << num << endl;
    return 0;
}
